frontend/src/admin/lessons/LessonItem.tsx

```diff
@@ -39,13 +43,24 @@ export default function LessonItem(props: {
           >
             <div className='word-group'>
               <h1 className='italic'>word</h1>
-              <h1 className='text-2xl text-purple-400 font-bold'>Valiant</h1>
+              <h1 className='text-2xl text-purple-400 font-bold'>
+                {props.lesson.word}
+              </h1>
             </div>
             <div className='word-group md:mt-0 xs:mt-4'>
               <h1 className='italic'>answer</h1>
-              <h1 className='text-2xl text-purple-400 font-semibold'>
-                Courageous
-              </h1>
+              {props.lesson.choices.map((choice:Choice) => {
+                if (choice.is_correct) {
+                  return (
+                    <h1
```

Comment:
Dont use `h1` here

Reply:
ah miss the changes removing the h1 tags are reflected in the branch feature/frontend-enhancements.. but yes, will remove this here also